lib/liquidTestRunner.js

```diff
@@ -326,13 +330,17 @@ async function runTestsWithOutput(
   firmId,
   handle,
   testName = "",
-  html_render = false
+  htmlInput = false,
+  htmlPreview = false
 ) {
   try {
-    const testRun = await runTests(firmId, handle, testName, html_render);
+    // console.log("htmlInput", htmlInput);
+    // console.log("htmlPreview", htmlPreview);
```

Comment:
Let's remove this debug variables

Reply:
console logs will be removed when I've finalized the dev